Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1533,18 +1553,18 @@ static void webProcessPoolHighDynamicRangeDidChangeCallback(CFNotificationCenter
 #if PLATFORM(MAC)
 void WebProcessPool::registerUserInstalledFonts(WebProcessProxy& process)
 {
-    if (m_userInstalledFontURLs) {
-        process.send(Messages::WebProcess::RegisterFontMap(*m_userInstalledFontURLs, *m_userInstalledFontFamilyMap,  sandboxExtensionsForFonts(*m_sandboxExtensionURLs, process.auditToken())), 0);
+    if (userInstalledFontURLs()) {
```

Comment:
We should capture it in local variable:
`if (auto fontURLs = userInstalledFontURLs()) { }`